pkgs/applications/accessibility/contrast/default.nix

```diff
@@ -0,0 +1,72 @@
+{ stdenv
+, fetchFromGitLab
+, cairo
+, cargo
+, dbus
+, desktop-file-utils
+, gettext
+, glib
+, gtk3
+, libhandy
+, meson
+, ninja
+, pango
+, pkgconfig
+, python3
+, rustc
+, rustPlatform
+, wrapGAppsHook
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "contrast";
+  version = "0.0.2";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    group = "World";
+    owner = "design";
+    repo = "contrast";
+    rev = version;
+    sha256 = "0rm705zrk9rfv31pwbqxrswi5v6vhnghxa8dgxjmcrh00l8dm6j9";
+  };
+
+  cargoSha256 = "06vgc89d93fhjcyy9d1v6lf8kr34pl5bbpwbv2jpfahpj9y84bgj";
+
+  nativeBuildInputs = [
+    desktop-file-utils
+    gettext
+    meson
+    ninja
+    pkgconfig
+    python3
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    cairo
+    dbus
+    glib
+    gtk3
+    libhandy
+    pango
+  ];
```

Comment:
Consider adding `hicolor-icon-theme` for the setup-hook?  Naturally I only discovered the problem because I had a /different/ package in my profile that also should have `hicolor-icon-theme` added :grin:

Reply:
This will no longer be necessary with https://github.com/NixOS/nixpkgs/pull/68552.

Reply:
we're going for #68552